src/richie/locale/es_ES/LC_MESSAGES/django.po

```diff
@@ -2,7 +2,7 @@ msgid ""
 msgstr ""
```

Comment:
Note that you will still need to enter these translations in Crowdin as populated .po files are pulled from there when building docker images.